.github/workflows/release.yml

```diff
@@ -49,13 +49,13 @@ jobs:
       - name: Set test env
         if: ${{ github.event.inputs.test != '' }}
         run: |
-          echo "::set-env name=AWS_ACCESS_KEY_ID::${{ secrets.PACKAGE_TEST_AWS_ACCESS_KEY_ID }}"
-          echo "::set-env name=AWS_SECRET_ACCESS_KEY::${{ secrets.PACKAGE_TEST_AWS_SECRET_ACCESS_KEY }}"
+          echo "AWS_ACCESS_KEY_ID=${{ secrets.PACKAGE_TEST_AWS_ACCESS_KEY_ID }}" >> $GITHUB_ENV
+          echo "AWS_SECRET_ACCESS_KEY=${{ secrets.PACKAGE_TEST_AWS_SECRET_ACCESS_KEY }}" >> $GITHUB_ENV
       - name: Set prod env
         if: ${{ github.event.inputs.test == '' }}
         run: |
-          echo "::set-env name=AWS_ACCESS_KEY_ID::${{ secrets.AWS_ACCESS_KEY_ID }}"
-          echo "::set-env name=AWS_SECRET_ACCESS_KEY::${{ secrets.AWS_SECRET_ACCESS_KEY }}"
```

Comment:
Dang, sorry that was my bad. I thought the ::set-env stuff was something I'd only experimented with in my test-release branch, I didn't know that had found its way into master :(

Reply:
No worries.